cluster/member_list.go

```diff
@@ -212,3 +213,13 @@ func groupMembersByKind(members []*Member) map[string][]*Member {
 	}
 	return groups
 }
+
+func (ml *MemberList) BroadcastEvent(message interface{}) {
```

Comment:
This function does not have the `includeSelf` parameter like the .NET counterpart. I don't think the .NET one is implemented correctly, as it compares the member id with actor system id, with former coming from cluster provider, and latter being a generated guid.